internal/service/vpc/resource_azure_privatelink_connection_approve_test.go

```diff
@@ -0,0 +1,162 @@
+package vpc_test
```

Comment:
I believe we don't add a suffix to the name of the test packages in this project

Reply:
1. It doesn't add changes to the provider.
2. vpc_test(s) are in a separate package because of cycle imports (the package already been there).

Reply:
> It doesn't add changes to the provider.

we try to list every change in the changelog, even when the docs were updated

the `no changelog` label should only be used, for example, when fixing a typo, or similar

this PR introduces new code so it definitely makes sense to add a changelog entry